### PR TITLE
MAINT: reconfigure wide-open nginx

### PR DESCRIPTION
**Pull request by eigenfoo**

### CODE REVIEW

File: nginx/default [hunk 1]
```diff
@@ -6,7 +6,35 @@ server {
     #server_name euphoria-recruiting.club www.euphoria-recruiting.club;
     server_name localhost;
     location /api/ {
-        # insert Apollo IP address and server port number below
+        if ($request_method = 'OPTIONS') {
+           add_header 'Access-Control-Allow-Origin' '*';
+           add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
```
Comment (luster) [inline nginx/default:11]: you should include OPTIONS and DELETE here i think

Comment (eigenfoo) [reply nginx/default:11]: Do you mean 

```suggestion
           add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS, PUT, DELETE';
```

?

That didn't work.

**Comment by luster** [comment]
Did you restart nginx and verify anything in the access or error log in nginx?

**Comment by eigenfoo** [comment]
@dhvanilshah helped us out with this. It turns out we just need to configure Apollo to accept and respond to a pre-flight `OPTIONS` request. It's all fixed in #127.

**Pull request closed without merging**